lib/deck.rb

```diff
@@ -0,0 +1,21 @@
+class Deck
+  attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    cards.length
+  end
+
+  def cards_in_category(category)
+    category_match = []
+    @cards.each do |card|
```

Comment:
Since you are not changing `@cards`, I would suggest leaving out the `@` sign here. Using `.each` won't mutate (change) the array, but other enumerable methods might so it's better to err on the side of caution.